src/containers/organizations/utils/IntegrationYamlUtils.js

```diff
@@ -118,6 +142,7 @@ export const exportTemplate = ({
   const fieldMappings = {
     [TEMPLATE_CONSTANTS.DATA_SQL_TYPE]: connectionString,
     [TEMPLATE_CONSTANTS.SQL_DRIVER_STRING]: driver,
+    [TEMPLATE_CONSTANTS.TABLE_LISTING_SQL]: tablesSql,
```

Comment:
I think this needs to be derived somehow?

Reply:
Derp, fixed!